Justfile

```diff
@@ -0,0 +1,15 @@
+# run tests/checks that are also run by github actions
+ci:
```

Comment:
Probably a bit "pedantic", but is this recipe CI specific? I would run this locally on my machine to check if everything is ok, right? Shall we name it `check` or `verify`?

Reply:
it depends on your point of view. My point of view is "execute exactly the actions executed by CI", that's why I like CI :)
And naming is hard here IMHO: `check` is a word that's used `cargo`, and this recipe does "a lot more" than `check`, so I'd like to avoid that.

Reply:
OK. Convinced. Let's keep `ci`.